### ISSUE TITLE
"Waiting for approval" status remains after request was approved

### ISSUE THREAD

**Issue by dara-meaney** [GoogleCloudPlatform/jit-access#425]
After a request has been approved. When i click on the status of the request through the original email, clicking on "View Request". I am still seeing the message "Waiting for approval". I can verify i have the role by checking the IAM page on GCP. If i was also to submit a new request, i can see the role  listed as "Active" with the remaining mins. 

**Steps to reproduce:** 
1. Open https://clover-dev-jit-access.uw.r.appspot.com/
2. Create a request for a role
3. Verify request email comes through
4. Get request approved
5. Go back to original email and click on "View Request"
6. You will see that the message still says "Waiting for approval" 

If the request requires three approvers, once all three have reviewed and approved the request. Is there a final email to suggest that the user has the role?

JIT Version 1.8

**Comment by jpassing** [comment]
> After a request has been approved. When i click on the status of the request through the original email, clicking on "View Request". I am still seeing the message "Waiting for approval"

Right, as long as the request is valid (based on `ACTIVATION_REQUEST_TIMEOUT`), the app doesn't check if the request was granted in the meantime and always displays "Waiting for approval". 

This behavior isn't ideal and would be good to change.

> If the request requires three approvers, once all three have reviewed and approved the request. Is there a final email to suggest that the user has the role?

When you request a role and select multiple approvers, then all these approvers get an email -- but it's sufficient if _one_ of them approves. Once the first approver approves, the role is granted and the requester gets an email.
